prow/apis/prowjobs/v1/types.go

```diff
@@ -189,6 +191,11 @@ type ProwJobSpec struct {
 	// https://github.com/tektoncd/pipeline
 	PipelineRunSpec *pipelinev1alpha1.PipelineRunSpec `json:"pipeline_run_spec,omitempty"`
 
+	// TektonPipelineRunSpec provides the basis for running the test as
+	// a pipeline-crd resource
+	// https://github.com/tektoncd/pipeline
+	TektonPipelineRunSpec *TektonPipelineRunSpec `json:"tekton_pipeline_run_spec,omitempty"`
```

Comment:
Is `PipelineRunSpec` different from `TektonPipelineRunSpec`?

Reply:
If its only for `v1beta1` then maybe we should add that in the comment

Reply:
`TektonPipelineRunSpec` is a struct that can contain different versions (currently beta1 but we could add v1 there)